PICMI_Python/particles.py

```diff
@@ -312,6 +312,12 @@ class PICMI_UniformFluxDistribution(_ClassWithInit):
 
     directed_velocity: vector of floats, default=[0.,0.,0.]
         Directed, average, velocity [m/s]
+
+    flux_tmin: float, optional
```

Comment:
Thanks! I'll approve as is. However, these names could be simply `tmin` and `tmax` to reduce the amount of typing.